src/interface/interface.json

```diff
@@ -186,6 +186,85 @@
                                 "profileSettingsLabel": "{String} profileSettingsLabel",
                                 "signOutLabel": "{String} signOutLabel"
                             }
+                        },
+                        "ProjectAccountSwitcher": {
+                            "methods": {
+                                "addAccount": {
+                                    "desc": "Pass in an instance of a ProjectAccountSwitcher Account",
+                                    "params": [
+                                        "{Account} [ReferenceAccount]",
+                                        "{Account} [ReferenceAccount] - (Optional) reference node for insertBefore"
+                                    ]
+                                },
+                                "addProject": {
+                                    "desc": "Pass in an instance of a ProjectAccountSwitcher Project",
+                                    "params": [
+                                        "{Project} [ReferenceProject]",
+                                        "{Project} [ReferenceProject] - (Optional) reference node for insertBefore"
+                                    ]
+                                },
+                                "open": {
+                                    "desc": "opens the profile"
+                                },
+                                "close": {
+                                    "desc": "closes the profile"
+                                },
+                                "setActive": {
+                                    "desc": "Pass an instance of either Account or Project",
+                                    "params": [
+                                        "{Account or Project} - The account or project currently chosen"
+                                    ]
+                                },
```

Comment:
But notice how the account image is wider: 
<img width="176" alt="screen shot 2017-05-30 at 4 06 55 pm" src="https://cloud.githubusercontent.com/assets/178653/26608745/109a7028-4552-11e7-9fbd-c732e64a890a.png">